H071221033/P10/final 1.sql

```diff
@@ -0,0 +1,62 @@
+#final 1
+# g = semua umur, pg = bo, pg13 = bo < 13th, r = restricted <17th (13-16), nc = >17
+
+
+# iqb, wali, dwn
+
+-- no1 iqb1
+SELECT f.title AS 'judulFilm', f.rating AS 'ratingFilm', GROUP_CONCAT(CONCAT(a.first_name, " ", a.last_name) SEPARATOR ', ') AS 'actors'
+FROM film f
+JOIN film_actor USING(film_id)
+JOIN actor a USING(actor_id)
+WHERE f.title LIKE 'w%' 
+GROUP BY f.title
```

Comment:
ada perbedaan sedikit atribut grouping luckily it's kind of the same thing so number 1 passed hurray!